public/components/trace_analytics/components/services/services_content.tsx

```diff
@@ -124,13 +123,17 @@ export function ServicesContent(props: ServicesProps) {
     );
 
     setIsServicesTableDataLoading(true);
-    handleServicesRequest(http, DSL, setTableItems, mode, dataSourceMDSId[0].id).finally(() =>
-      setIsServicesTableDataLoading(false)
-    );
+    handleServicesRequest(
+      http,
+      DSL,
+      setTableItems,
+      mode,
+      setServiceMap,
+      dataSourceMDSId[0].id
+    ).finally(() => setIsServicesTableDataLoading(false));
 
     setIsServicesDataLoading(true);
     Promise.all([
-      handleServiceMapRequest(http, serviceMapDSL, mode, dataSourceMDSId[0].id, setServiceMap),
       isServiceTrendEnabled
         ? handleServiceTrendsRequest(http, '1h', setServiceTrends, mode, [], dataSourceMDSId[0].id)
         : Promise.resolve(null),
```

Comment:
same here, it can just be if isServiceTrendEnabled, then `handleServiceTrendsRequest(http, '1h', setServiceTrends, mode, [], dataSourceMDSId[0].id).finally(() => setIsServicesDataLoading(false))`

Reply:
Adjusted too
```
    setIsServicesDataLoading(true);
    if (isServiceTrendEnabled) {
      handleServiceTrendsRequest(http, '1h', setServiceTrends, mode, [], dataSourceMDSId[0].id)
        .finally(() => setIsServicesDataLoading(false));
    } else {
      setIsServicesDataLoading(false);
    }
```
Thanks